_issues/but-she-isnt-trustworthy.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Hillary Clinton’s not trustworthy. She lies all the time, right?"
+categories: deception
+---
+
+Q: Hillary Clinton’s not trustworthy. She lies all the time, right?
+
+A: No. Hillary is actually generally one of the most truthful
+politicians to have been in the presidential races for the past 9
+years!
+
+
+* Politifact.org - an independent fact-checking organization that was
+  winner of the Pulitzer prize in journalism has found that Hillary is
+  second best overall in a field of 20 presidential candidates since
+  2007 (the last 9 years). See
+  [a graph for yourself here](https://datavizblog.com/2016/07/24/political-dataviz-who-lies-more-a-comparison-robert-mann/)
+
+* Specifically for 2016, here is Hillary Clinton vs Donald Trump in
+  [direct comparison](http://www.politifact.com/truth-o-meter/lists/people/comparing-hillary-clinton-donald-trump-truth-o-met/)
+  for truthfulness.
+
+* And another
+  [great write-up by Politifact](http://www.politifact.com/truth-o-meter/article/2016/jun/29/fact-checking-2016-clinton-trump/),
+  which talks about how Politifact performs its review of both Hillary
```

Comment:
prefer "which discusses the methodology of how Politifact performs its reviews of both Hillary and Trump".